backend/src/v2/driver/driver.go

```diff
@@ -805,19 +805,51 @@ func extendPodSpecPatch(
 			if toleration != nil {
 				k8sToleration := &k8score.Toleration{}
 				if toleration.TolerationJson != nil {
-					err := resolveK8sJsonParameter(ctx, opts, dag, pipeline, mlmd,
-						toleration.GetTolerationJson(), inputParams, k8sToleration)
+					resolvedParam, err := resolveInputParameter(ctx, dag, pipeline, opts, mlmd,
+						toleration.GetTolerationJson(), inputParams)
 					if err != nil {
-						return fmt.Errorf("failed to resolve toleration: %w", err)
+						return fmt.Errorf("failed to resolve k8s parameter: %w", err)
```

Comment:
I preferred the old log message.

Reply:
whoops that wasn't suppose to change, fixed